governance/roles.md

```diff
@@ -92,23 +93,38 @@ TLs are assigned by CNCF Technical Oversight Committee
 
 ## Role of project leads
 
-Project Leads will lead larger streams of work that require sustained
-effort and coordination.  In some cases, these projects remain ongoing.
+Project Leads lead specific projects where a deliverable has been defined 
+within the issue.
 
 Project Leads are nominated and approved by the following process:
-  1. Project Lead actively participates in the group, making a proposal or
-  volunteering to take on a project that has been prioritized by the group
+  1. Project Lead actively participates in the group, initiates the proposal or
+  volunteers to take on a project that has been prioritized by the group
   1. A Chair or TL nominates a candidate.
-  1. The nomination is communicated via a pull request annotating the list of members in the 
-  [SIG README](/README.md) with a link to the issue tracking the project. 
+  1. The nomination is communicated on the issue tracking the project 
   The nomination is typically open for a week (but may be shorter with LGTM of at least two Chairs).
-  1. Members are encouraged to review the tracking issue and work together
-  to ensure that the Project Lead is set up for success or suggest alternatives.
+  1. The issue is updated with the assignment of the project lead as "assignee" alongside the SIG Leadership member.
+  1. Project Leads will be given the OWNER role of the directory or sub-directory for their team's activities in order to empower merge/approve.
 
 Depending on the expected length of the project, Project Leads may be assigned
 as OWNERS of the project directory in this repository to help facilitate and moderate new content to the
 project.
 
+## Role of Team Leads
+
+Team Leads lead larger streams of work that require sustained effort and 
+coordination and remain ongoing.
```

Comment:
Note that an on-going project may not imply a team.  For example, the Security Reviews initially kicked off with a facilitator role and a series of projects (one for each review) where each was executed by a different group of volunteers.  Even though there was intentional overlap in the members on each project, we did not form a team to ensure sustainability of the initiative.

Reply:
Also, teams must have a chair sponsor.  (At least that's what we've done so far with Policy and Security Reviews, where the later isn't technically a team, but we wanted to ensure support for that and added chair review/approval as part of the process.  I think even if someone has OWNER role in a directory, if they are publishing a whitepaper or other artifact likely to have external readers, like a Security Review, then chair approval should be required). We could extend this to Tech Leads, but need a bit of discussion on the leadership team before doing so)